var/spack/repos/builtin/packages/qt-base/package.py

```diff
@@ -91,6 +99,8 @@ class QtBase(QtPackage):
     url = QtPackage.get_url(__qualname__)
```

Comment:
What would you think of this?
```suggestion
    homepage = QtPackage.get_url("QtCore")
    url = QtPackage.get_url(__qualname__)
```
Maybe similar for qt-declarative?

Reply:
You meant `homepage = QtPackage.get_homepage("QtCore")`, which would expand to `https://doc.qt.io/qt-6/qtcore-index.html`?
Yes, it  would make sense to link to the most important module of each package. Way more useful than the github repo pages, e.g.  `https://github.com/qt/qtbase`!

Reply:
Hi, is this still a draft (not ready for merge) or is it good to merge?